src/models/streaming/SessionConfig.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Configuration for streaming job
+ */
+export class SessionConfig {
+    metadata?: string;
+
+    /**
+     * @param metadata (Optional) metadata to be associated with the streaming job
+     */
+    constructor(
+        metadata?: string
+    ) {
```

Comment:
is this how it's supposed to be formatted in JS?

Reply:
yes, we run a 3rd party linter on the project as part of the travis ci pipeline and this is what it accepts :)